tasks/move-granules/tests/test-index.js

```diff
@@ -128,7 +125,7 @@ test.beforeEach(async (t) => {
   );
 
   const payloadPath = path.join(__dirname, 'data', 'payload.json');
-  const rawPayload = await readFile(payloadPath, 'utf8');
+  const rawPayload = fs.readFileSync(payloadPath, 'utf8');
```

Comment:
I'm not sure I understand what this change is fixing.

Reply:
Just a minor cleanup. Everywhere else is using `fs.readFileSync` rather than importing `promisify` and using it with `fs.readFile`, so I did it for consistency to and to eliminate the `promisify` import.